### ISSUE TITLE
`@stylistic/selector-pseudo-class-parentheses-space-inside` keeps reporting a pseudo-class it has already fixed, when the last argument is a comment

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#128]
Found while reviewing the fix for #123, which does not touch this and does not cause it — the behaviour is the same on either side of that commit.

### What minimal example or steps are needed to reproduce the bug?

```css
a:not(/**/) {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/selector-pseudo-class-parentheses-space-inside": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### Which Stylelint-related dependencies are you using?

```json
{
  "@stylistic/stylelint-plugin": "5.3.0",
  "stylelint": "17.14.1"
}
```

### What did you expect to happen?

`a:not( /**/ ) {}`, and a second run over that output reporting nothing.

### What actually happened?

The output is right. The warning is not:

```
1:11  Expected single space before ")" (@stylistic/selector-pseudo-class-parentheses-space-inside)
```

The fixer does its job and is idempotent — `a:not( /**/ ) {}` comes out of the first run and never changes again. The rule then reports that output every time it is linted, at a column that falls inside the comment rather than on the parenthesis. So `--fix` leaves a warning standing on code it has already made correct, and no amount of running it clears it.

It happens under `always` only, and only when the pseudo-class's last argument holds nothing but comments:

```css
/* reported forever */   a:not(/**/) {}
/* reported forever */   a:not(b, /**/) {}
/* reported forever */   a:not(/**/ /**/) {}

/* settles */            a:not(b /**/) {}
/* settles */            a:not(/**/ b) {}
/* settles */            a:not(/**/, b) {}
```

`never` has the same hole seen from the other side — it reports nothing at all where it should:

```css
/* a space before ")", no warning */   a:not(/**/ ) {}
/* the same space, reported at 1:13 */ a:not(b /**/ ) {}
```

### Do you have a proposal to fix the bug?

The whitespace between such a comment and the closing parenthesis is not in the parsed tree at all. `postcss-selector-parser` drops it on a plain round-trip, without anyone touching the result:

```js
parser((root) => root).processSync(`a:not( /**/ )`)   // 'a:not( /**/)'
parser((root) => root).processSync(`a:not( b /**/ )`) // 'a:not( b /**/ )'  ← survives
```

It survives in the second case because the comment is folded into the raws of the node in front of it, which is what #123 was about. An argument made only of comments has no node to fold into, so the parser puts the run in the container's own spaces and `toString()` never prints it back.

That is why `paramString` for `a:not( /**/ )` is `" /**/"`. `prevCharIsSpace` can never become `true`, so `always` reports forever and `never` never reports. The reported index is short by the width of the missing space, which is how it ends up inside the comment.

The root cause is in the dependency, so the honest fix is upstream. A rule-level guard is possible in the meantime — recognising that the last node is a comment and neither reporting nor fixing that side, as the rule already declines the closing side of a multiline parameter list under `never` — which trades a false positive for a blind spot that `never` already has.

Whichever way it goes, it should be pinned by a `reject` case that declares `fixed`, since that is what makes `testRule` relint the output and refuse a warning that outlives its own fix.
